### ISSUE TITLE
Logger log method ignore the log level

### ISSUE THREAD

**Issue by tiyilon** [phalcon/cphalcon#10593]
Logger log method ignore the log level ,as follow code:

```php
public function testAction(){

	$loggerPath = "../app/logs/log_test";
	$logger = new FileLogger($loggerPath);


	$logger->setLogLevel( Logger:: WARNING ) ;

	$logger->log("test");
	$logger->warning("warn");
	$logger->error("error!");
	$logger->debug("debug");

}
```

the result in log file :+1: 

```
[Mon, 06 Jul 15 16:19:13 +0800][DEBUG] test
[Mon, 06 Jul 15 16:19:13 +0800][WARNING] warn
[Mon, 06 Jul 15 16:19:13 +0800][ERROR] error!
```

why log method output the log ([DEBUG] test) but the debug method did not ?

Phalcon version :  2.0.3

**Comment by SidRoberts** [comment]
This should be fixed in #10594. :smile:

**Comment by andresgutierrez** [comment]
Fixed in the 2.0.x branch
